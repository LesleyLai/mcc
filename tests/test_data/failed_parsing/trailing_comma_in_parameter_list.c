int foo(int a, int b, int c,) {
  return a + b + c;
}

int main(void) {
  return foo(1, 2, 3);
}
