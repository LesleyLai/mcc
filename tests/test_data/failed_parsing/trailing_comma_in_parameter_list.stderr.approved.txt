{{filename}}:1:29: Error: Expect parameter declarator
1 | int foo(int a, int b, int c,) {
  |                             ^

